mod.js

```diff
@@ -173,39 +171,36 @@ class Node1 {
         }
         c = new Node1({
           path,
-          func,
+          handler,
         });
         n.children.set(path[0], c);
         n = c;
-      } else if (func) {
-        if (n.func) {
-          throw new Error(
-            `a function is already registered for path "${path}"`,
-          );
+      } else if (handler) {
+        if (n.handler) {
+          throw new Error(`a handler is already registered for path "${path}"`);
         }
-        n.func = func;
+        n.handler = handler;
       }
       break;
     }
     return n;
   };
-  #insert = (path, func) => {
+  #insert = (path, handler) => {
     let n = this;
     let c = n.children.get(path[0]);
     if (c) {
-      n = c.#merge(path, func);
+      n = c.#merge(path, handler);
     } else {
       c = new Node1({
         path,
-        func,
+        handler,
       });
       n.children.set(path[0], c);
       n = c;
     }
     return n;
   };
 }
-export { Node1 as Node };
```

Comment:
Is this file generated by `npm run build`, why is the export statement deleted?

Reply:
Yes, it's generated by `npm run build`. I don't know why but I added it again.